lib/ansible/galaxy/role.py

```diff
@@ -42,6 +42,22 @@
 display = Display()
 
 
+def _is_tar_member_path_sane(path):
+    """
+    Check if the given path from a tar member is safe to extract.
+
+    This method looks for common risks that can lure in the path:
+    - Absolute paths that allow to create files outside of the target.
+    - `..` parts that allow to write files outside of the target.
```

Comment:
If we ever start generating the docs from docstrings, this will be problematic — RST uses double backticks for inline code, unlike Markdown.